src/adr/articles/0009-integration-ualpa.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: layouts/adr.njk
+title: "Integration with Use a LPA"
+weight: 1
+date: 2022-03-16
+review_in: 12 months
+tags:  
+    - adr
+    - open_standards
+    - apis
+    - language
+    - integration
+areas_of_coverage: ["Digital Service"]
+status: "accepted"
+contributors: ["John Nolan"]
+adr_number: "0009"
+---
+
+## Context
+
+Our [Use a LPA service](https://use-lasting-power-of-attorney.service.gov.uk/home) is the latest service built by the OPG.
```

Comment:
We shouldn't link directly to the service, we should link to the gov.uk start page